laplace/subnetlaplace.py

```diff
@@ -0,0 +1,118 @@
+import torch
+
+from laplace.baselaplace import FullLaplace, DiagLaplace
+
+from laplace.curvature import BackPackGGN
+from laplace.subnetmask import LargestVarianceDiagLaplaceSubnetMask
+
+
+__all__ = ['FullSubnetLaplace']
+
+
+class FullSubnetLaplace(FullLaplace):
```

Comment:
Since we have just one now, I would name it `SubnetLaplace`. The most similar method is probably `LowRankLaplace` and also is implicitly _Full_.

Reply:
`'full'` in the key below is necessary though, as also in `LowRank`.

Reply:
Done